app/views/rooms/show.html.erb

```diff
@@ -5,59 +5,59 @@
-        <% if @room.closed? %>
-          <h4>This thread has been closed.</h4>
-          <h5>No new messages may be posted.</h5>
-        <% else %>
-          <%= form_for :message, url: messages_path do |f| %>
-            <%= f.text_field :body %>
-            <%= hidden_field_tag "room_id", @room.id %>
-            <%= f.submit "Create message", class: "btn col m6 s12" %>
+        <div id="new-message" class="input-field col s12">
+          <% if @room.closed? %>
+            <h4>This thread has been closed.</h4>
+            <h5>No new messages may be posted.</h5>
+          <% else %>
+            <%= form_for :message, url: messages_path do |f| %>
+              <%= f.text_field :body %>
+              <%= hidden_field_tag "room_id", @room.id %>
+              <%= f.submit "Create message", class: "btn col m6 s12" %>
+            <% end %>
           <% end %>
-        <% end %>
-      </div> <!-- new-message -->
-    </div> <!-- messages -->
+        </div> <!-- new-message -->
+      </div> <!-- messages -->
 
-    <div class="col m3 offset-m1 s12 room-details">
-      <div class="row">
-        <div class="room-status">
-          <p class="center-align">Room status: <b><%= @room.display_current_state %></b></p>
-          <% if current_user == @room.user && !@room.closed? %>
-            <%= button_to "Close this Topic", room_path(@room), method: :put, class: "btn col s12" %>
-          <% end %>
-        </div> <!-- room-status -->
-      </div> <!-- row -->
+      <div class="col m3 offset-m1 s12 room-details">
+        <div class="row">
+          <div class="room-status">
+            <p class="center-align">Room status: <b><%= @room.display_current_state %></b></p>
+            <% if !@room.closed? %>
+              <%= current_user.close_topic(@room) %>
+            <% end %>
+          </div> <!-- room-status -->
+        </div> <!-- row -->
 
-      <div id="choices">
-        <h4>Choices:</h4>
-        <% @room.choices.each do |choice| %>
-          <div class="<%=choice.title%>">
-            <% if @room.closed? %>
-              <div class="<%=choice.chosen_display%>">
+        <div id="choices">
+          <h4>Choices:</h4>
+          <% @room.choices.each do |choice| %>
+            <div class="<%=choice.title%>">
+              <% if @room.closed? %>
+                <div class="<%=choice.chosen_display%>">
+                  <h5><%= choice.title %></h5>
+                </div>
+              <% else %>
                 <h5><%= choice.title %></h5>
-              </div>
-            <% else %>
-              <h5><%= choice.title %></h5>
-            <% end %>
-          </div>
-        <% end %>
-      </div> <!-- choices -->
-    </div> <!-- room-details -->
+              <% end %>
+            </div>
+          <% end %>
+        </div> <!-- choices -->
+      </div> <!-- room-details -->
 
-  </div> <!-- row -->
-</div> <!-- container -->
+    </div> <!-- row -->
+  </div> <!-- container -->
```

Comment:
Oh yeah I remember when Mejia showed us this I wasn't using vim yet. Dope thanks for the resource.